qt_dotgraph/src/qt_dotgraph/pydotfactory.py

```diff
@@ -175,4 +174,4 @@ def create_dot(self, graph):
         if type(dot) != str:
             dot = dot.decode()
         # sadly pydot generates line wraps cutting between numbers
-        return dot.replace("\\\n", "")
+        return dot.replace('\\\n', '')
```

Comment:
Was this done by an auto-format tool or to pass linting tests? Double to single quotes seems like a unnecessary change. If this is to make the linter happy, can we just disable that rule?

Reply:
In order to pass flake8 double quotes like these need to be changed to single quotes.

Reply:
Probably shouldn't be in this PR though.